tephra-core/src/main/java/co/cask/tephra/util/TxUtils.java

```diff
@@ -88,4 +92,12 @@ public static long getFirstShortInProgress(Map<Long, TransactionManager.InProgre
     }
     return firstShort;
   }
+
+  /**
+   * Returns the timestamp for calculating time to live for the given cell timestamp.
+   * This takes into account pre-existing non-transactional cells while calculating the time.
+   */
+  public static long getTtlTimestamp(long cellTs) {
```

Comment:
`getTTLTimestamp()` is also hard to read. I'll change it to `getTimestampForTTL()`.